lib/ansible/modules/files/template.py

```diff
@@ -17,6 +17,7 @@
 ---
 module: template
 version_added: historical
+short_description: Create a files and directories by using templates.
```

Comment:
SHould just be what was on the original `short_description` in the template_common file.